src/current/v26.1/ory-overview.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Ory Overview
+summary: Learn about Ory and its component services - Hydra, Kratos, and Keto
+toc: true
+docs_area: Integrate
+---
+
+Ory is an open-source identity and access management (IAM) platform that provides modular components for authentication and authorization in distributed systems. Key components include:
+
+- [Ory Hydra](https://www.ory.sh/hydra) for OAuth2 and OIDC flows.
+- [Ory Kratos](https://www.ory.sh/kratos) for identity management (including users, groups, and organizations).
+- [Ory Keto](https://www.ory.sh/keto) for fine-grained authorization and relationship-based access control (ReBAC,  inspired by Google Zanzibar).
+
+<img src="{{ 'images/v26.1/integrate-ory-architecture-overview.png' | relative_url }}" alt="Ory Architecture Overview"  style="border:1px solid #eee;max-width:80%;margin:auto;display:block" />
+
+## Ory components
+
+### Ory Hydra
+
+Ory Hydra is a server implementation of the OAuth 2.0 authorization framework and the OpenID Connect Core 1.0. It tracks clients, consent requests, and tokens with strong consistency to prevent replay attacks and duplicate authorizations.
+
+The OAuth 2.0 authorization framework enables a third-party application to obtain limited access to an HTTP service, either on behalf of a resource owner by orchestrating an approval interaction between the resource owner and the HTTP service, or by allowing the third-party application to obtain access on its own behalf.
+
+<img src="{{ 'images/v26.1/integrate-ory-oauth2-flow.png' | relative_url }}" alt="OAuth2 Flow"  style="border:1px solid #eee;max-width:80%;margin:auto;display:block" />
+
+The OAuth 2.0 authorization flow involving a client application, the resource owner, Ory Hydra (as the authorization server), and the resource server is structured as follows:
+
+<img src="{{ 'images/v26.1/integrate-ory-hydra-flow.png' | relative_url }}" alt="Interaction flow using Ory Hydra"  style="border:1px solid #eee;max-width:80%;margin:auto;display:block" />
+
+The sequence diagram depicts the interactions between four key components:
+
+- the client
```

Comment:
suggest using exact same capitalized spelling of terms as in the diagram for easier scan/pattern match by reader, e.g.

- Client
- Resource Owner ..
- Ory Hydra
- Resource Server